x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_form_consumer_selection.tsx

```diff
@@ -115,8 +123,8 @@ export const RuleFormConsumerSelection = (props: RuleFormConsumerSelectionProps)
     }, [consumers]);
 
   useEffect(() => {
-    // At initialization we set to NULL to know that nobody selected anything
-    onChange(null);
+    // At initialization, select the first value
```

Comment:
Since `initialSelectedConsumer?: RuleCreationValidConsumer | null;` can be undefined or null. Do you think it will be a good idea to do something like that to make sure that `STACK_ALERTS_FEATURE_ID` is selected if a dev forget to initialize the value

```ts
 useEffect(() => {
    // At initialization, select the first value
    if (!validatedSelectedConsumer) {
      if(consumers.includes(STACK_ALERTS_FEATURE_ID) {
          onChange(STACK_ALERTS_FEATURE_ID);
          return;
      }
       onChange(consumers[0] as RuleCreationValidConsumer);
    }
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, []);
```